src/data/test_sqlite.py

```diff
@@ -0,0 +1,17 @@
+from sqlite_converter import SQLiteDataConverter
```

Comment:
Fantastic with a test script! I would move it to a separate `test` directory, write it as a unit test, and package it with minimal data files required for testing -- but let's leave that till after this PR.

Reply:
Agreed. I did it this way because you cannot import from .py-files that are not in the same folder as the test scripts - unless it is installed as a package, which I did not want to mess with right now.


Rasmus